src/content/events/2024/hackathon-march-2024/germany-heidelberg.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Hackathon - March 2024 (Heidelberg)
+subtitle: Local node of the nf-core hackathon at GHGA, Heidelberg.
+type: hackathon
+start_date: '2024-03-18'
+start_time: '10:00+02:00'
+end_date: '2024-03-20'
+end_time: '18:00+02:00'
+location_name: Heidelberh
+address: .., .. Heidelberg, Germany
+location_url: https://www.ghga.de/events
+location_latlng: [ , ]
```

Comment:
```suggestion
# location_latlng: [ , ]
```
I am afraid the site will not build with empty location coordinates, better to comment it out.

Reply:
ach I see